src/parse-payload.ts

```diff
@@ -1,14 +1,14 @@
 import { InvocationPayload } from "./types.ts";
+import { readAll } from "./deps.ts";
 
 // Inject the readAll method dependency for easier testing
 export const ParsePayload = async (
-  // deno-lint-ignore no-explicit-any
-  readAll: (input: any) => Promise<ArrayBuffer>,
+  readStdin: typeof readAll,
```

Comment:
```
export async function readAll(r: Deno.Reader): Promise<Uint8Array>
```

🔗 https://deno.land/std@0.99.0/io/util.ts

This appears to be more strict typing overall! It's nice to use what the standard library provides!